src/js/media-chrome-listitem.js

```diff
@@ -0,0 +1,106 @@
+import { MediaUIAttributes } from './constants.js';
+import { defineCustomElement } from './utils/defineCustomElement.js';
+import {
+  Window as window,
+  Document as document,
+} from './utils/server-safe-globals.js';
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+<style>
+  :host {
+    display: list-item;
+  }
+</style>
+<li>
+  <slot></slot>
+</li>
+`;
+
+class MediaChromeListitem extends window.HTMLElement {
+  static get observedAttributes() {
+    return ['disabled', 'aria-selected', MediaUIAttributes.MEDIA_CONTROLLER];
+  }
+
+  constructor(options = {}) {
+    super();
+
+    const shadow = this.attachShadow({ mode: 'open' });
+
+    const listitemHTML = template.content.cloneNode(true);
+    this.nativeEl = listitemHTML;
+
+    shadow.appendChild(listitemHTML);
+  }
+
+  enable() {
+    if (!this.hasAttribute('tabindex')) {
+      this.setAttribute('tabindex', -1);
+    }
+    if (!this.hasAttribute('aria-selected')) {
+      this.setAttribute('aria-selected', "false");
+    }
+  }
+
+  disable() {
+    this.removeAttribute('tabindex');
+  }
+
+  attributeChangedCallback(attrName, oldValue, newValue) {
+    if (attrName === MediaUIAttributes.MEDIA_CONTROLLER) {
+      if (oldValue) {
+        const mediaControllerEl = document.getElementById(oldValue);
+        mediaControllerEl?.unassociateElement?.(this);
+      }
+      if (newValue) {
+        const mediaControllerEl = document.getElementById(newValue);
+        mediaControllerEl?.associateElement?.(this);
+      }
+    } else if (attrName === 'disabled' && newValue !== oldValue) {
+      if (newValue == null) {
+        this.enable();
+      } else {
+        this.disable();
+      }
+    }
+  }
+
+  connectedCallback() {
+    if (!this.hasAttribute('disabled')) {
+      this.enable();
+    }
+
+    this.setAttribute('role', 'option');
+
+    const mediaControllerId = this.getAttribute(
+      MediaUIAttributes.MEDIA_CONTROLLER
+    );
+    if (mediaControllerId) {
+      const mediaControllerEl = document.getElementById(mediaControllerId);
+      mediaControllerEl?.associateElement?.(this);
+    }
+  }
+
+  disconnectedCallback() {
+    this.disable();
+
+    const mediaControllerId = this.getAttribute(
+      MediaUIAttributes.MEDIA_CONTROLLER
+    );
+    if (mediaControllerId) {
+      const mediaControllerEl = document.getElementById(mediaControllerId);
+      mediaControllerEl?.unassociateElement?.(this);
+    }
+  }
+
+  get keysUsed() {
+    return ['Enter', ' '];
```

Comment:
Does it use these? Not totally familiar with how keysUsed is used, but this feels handled by listbox.